src/main/scala/org/apache/spark/sql/kinesis/KinesisReader.scala

```diff
@@ -155,10 +155,6 @@ private[kinesis] case class KinesisReader(
         s"Describe Streams") {
         try {
           getAmazonClient.describeStream(describeStreamRequest)
-        } catch {
```

Comment:
Why remove this catch clause?

Reply:
We are already making describeStream API call within` retryOrTimeout` block which takes care of `LimitExceededException`. If the API call fails with `AccessDeniedException` or any other invalid credentials exception, we should error out and stop the streaming application.  Earlier we were absorbing the exceptions which was given a false impression that streaming job is fine.  See #10 and #12